Document advertising additional startd attributes (SOFTWARE-5324) (#134)
osg-htc/docs: docs/resource-sharing/os-backfill-containers.md

    Additionally, the processes that manage jobs also use some amount of memory.
    Therefore, it is important to give the container some extra room.

### Advanced: Advertising additional pilot attributes

> You can put arbitrary additional attributes in the machine ads that you advertise to the OSPool.

Reviewer: ```suggestion
You can put arbitrary additional attributes in the machine ads that the glidein advertises to the OSPool.
```

Author: We don't call them glideins unless they're submitted by a factory but I can use "the pilot advertises" instead

> You can put arbitrary additional attributes in the machine ads that you advertise to the OSPool.

Author: ```suggestion
You can put arbitrary additional attributes in the machine ads that the pilot advertises to the OSPool.
```

> These attributes will show up when you do `condor_status -l` on your machine.

Reviewer: ```suggestion
These attributes will show up when users run `condor_status -l` against your glidein.
```

I think it's safe to assume that some of these sites don't have HTCondor client tools

> These attributes will show up when you do `condor_status -l` on your machine.

Author: ```suggestion
These attributes will show up when users run `condor_status -l` against your pilot.
```

This could be useful for advertising something about the way the pilot was provisioned.
To do this, volume-mount a file containing `key=value` pairs to `/etc/osg/extra-attributes.cfg`.
Keys must be valid classad attribute names and values must be valid classad expressions.
Multi-line strings are not supported.
A line starting with `#` will be treated as a comment.
For example:

```
# The Kubernetes namespace this pod is running under

> Kubernetes_Namespace = "path-osgdev"

Reviewer: ```suggestion
KUBERNETES_NAMESPACE = "path-osgdev"
```

Unless there's some convention for these attrs that indicate that we should use title casing

# The deployment for this pilot

> Kubernetes_Deployment = "osgvo-docker-pilot-gpu"

Reviewer: ```suggestion
KUBERNETES_DEPLOYMENT = "osgvo-docker-pilot-gpu"
```
